### PR TITLE
Java

### PR DESCRIPTION
**Pull request by sangnie**

### CODE REVIEW

File: benchmarks/web-search/server/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM cloudsuite/spark-base:base
+FROM ubuntu:14.04
```
Comment (neo-apz) [inline benchmarks/web-search/server/Dockerfile]: Shouldn't this be `java`?

**Comment by neo-apz** [comment]
The `java` dockerfile has to be under the commons folder. More specifically, there should be a `java` folder under the commons folder, and another folder inside the java folder with the version number (like spark).

**Comment by neo-apz** [comment]
You didn't move the dockerfile to the java folder; you created a new one.
Please remove the extra one.

**Comment by ustiugov** [comment]
hey Arash! what should we do with this PR?

**Comment by neo-apz** [comment]
I'm merging this. The only two unsuccessful builds are related to Hadoop (see #118).

**Pull request merged**